web/packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -58,92 +51,97 @@ declare global {
   }
 }
 
+const invalidStateMessage =
```

Comment:
I don't know how I feel about having the product say "something went wrong, please file a bug report." Can we just display a more generic error and log whatever information we can?

Reply:
Changed to `const invalidStateMessage = 'internal application error';` in 6cfb564322a1deedff79e1451b2d5982d8f92bbe